src/main/scala/uk/gov/hmrc/govukfrontend/views/Utils.scala

```diff
@@ -30,6 +34,10 @@ trait Utils extends UtilsSupport {
       .map(p => s"$p/$file")
       .getOrElse(uk.gov.hmrc.hmrcfrontend.controllers.routes.Assets.at(s"govuk/$file").url)
 
+  private[views] def generateI18TranslationAttributes(strings: Option[TranslatableStrings], attributeName: String) =
+    strings.getOrElse(Map.empty).map { case (k, v) =>
+      Html(s"""$i18nDataAttribute.$attributeName.${HtmlFormat.escape(k)}="${HtmlFormat.escape(v)}" """)
+    }
```

Comment:
@kyle-bowden could we copy the name / params from https://github.com/alphagov/govuk-frontend/blob/d92a05aeea22236c00760feb5289f450ab9035c6/src/govuk/macros/i18n.njk#L13-L15 and I reckon rather than having a private i18nDataAttribute just have the data-i18n inline below because it's used once